src/utils/inference/markupInference.ts

```diff
@@ -155,7 +158,7 @@ function setCommonAttributes(common: Element, items: Element[]) {
 function ignoreDivChildNode(node: Node): boolean {
   return (
     NON_RENDERED_NODES.includes(node.nodeType) ||
-    (node.nodeType === Node.TEXT_NODE && node.textContent.trim() === "")
+    (node.nodeType === Node.TEXT_NODE && node.textContent?.trim() === "")
```

Comment:
I think this isn't handling properly when `textContent` is `null`. This will return `false`, because `null !== ""`, but I think it should return true for a node with type text where the content is `null`, as well as empty string.

I think it should be:
```suggestion
    (node.nodeType === Node.TEXT_NODE &&
      (node.textContent == null || node.textContent.trim() === "")
    )
```

Reply:
Great catch